pages/kieslerkreis/index.js

```diff
@@ -0,0 +1,103 @@
+import Header from "@/components/Header";
+import KieslerKreis from "@/components/KieslerKreis";
+import { styled } from "styled-components";
+
+const KieslerkreisTextArticle = styled.article`
+  padding: 15px;
+`;
+
+const KieslerkreisTextSection = styled.section`
+  padding-bottom: 20px;
+  line-height: 1.5em;
+`;
+
+export default function KieslerkreisPage({ allEntries }) {
+  const behaviorKieslerkreisArray = allEntries
+    .map((entry) => entry.behaviorKieslerkreis)
+    .filter((value) => value !== undefined)
+    .map((value) => JSON.parse(value));
+  const behaviorChangeKieslerkreisArray = allEntries
+    .map((entry) => entry.behaviorChangeKieslerkreis)
+    .filter((value) => value !== undefined)
+    .map((value) => JSON.parse(value));
+
+  function averageArrays(arrays) {
+    const result = [];
+    const length = 8;
+    for (let i = 0; i < length; i++) {
+      let sum = 0;
+      let count = 0;
+      for (const array of arrays) {
+        if (array[i] !== null) {
+          sum += array[i];
+          count++;
+        }
+      }
+      result.push(count === 0 ? 0 : Math.round(sum / count));
+    }
+    return result;
+  }
+  const overviewDataset = {
+    averageBehaviorKieslerkreis: averageArrays(behaviorKieslerkreisArray),
+    averageBehaviorChangeKieslerkreis: averageArrays(
+      behaviorChangeKieslerkreisArray
+    ),
+  };
+  console.log(overviewDataset);
```

Comment:
There is a console.log left.

Reply:
Also have a look at the console.log in newanalysis/index.js line 96.

Reply:
thanks! 👀